boards/microbit-v2/doc.txt

```diff
@@ -0,0 +1,36 @@
+/**
+@defgroup    boards_microbit_v2 BBC micro:bit v2
+@ingroup     boards
+@brief       Support for the BBC micro:bit v2
+
+## Overview
+
+The [micro:bit v2](https://www.microbit.co.uk/) was designed by the BBC and
+released in 2020.
+
+The board is based on the Nordic nRF52833 SoC, featuring 128KiB of RAM, 512KiB
+of ROM, and a 2.4GHz radio, that supports Bluetooth Low Energy (BLE), 802.15.4
+as well as a Nordic proprietary radio mode.
+
+Additionally the boards features 2 buttons, a 5x5 LED matrix, a speaker, a
+microphone, an accelerometer and a magnetometer.
+
+##  Flashing and Debugging
```

Comment:
Does the board come with a USB bootloader preinstalled?

Reply:
There's separate programmer chip with DAPLink, similar to v1. That's why we can use openocd and pyocd and do on-chip debugging directly via USB. The programmer chip can also be reflashed with a JLink compatible firmware but since it's not the default, I didn't document it here.